test/integration/server/ParametersTest.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) 2023 Vaticle
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package com.vaticle.typedb.core.server;
+
+import com.vaticle.typedb.core.common.exception.TypeDBCheckedException;
+import com.vaticle.typedb.core.server.parameters.CoreConfig;
+import com.vaticle.typedb.core.server.parameters.CoreConfigFactory;
+import com.vaticle.typedb.core.server.parameters.CoreConfigParser;
+import com.vaticle.typedb.core.server.parameters.util.Option;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static com.vaticle.typedb.common.collection.Collections.set;
+import static com.vaticle.typedb.core.server.common.Constants.TYPEDB_LOG_ARCHIVE_EXT;
+import static com.vaticle.typedb.core.test.integration.util.Util.deleteDirectory;
+import static org.junit.Assert.assertEquals;
+
+public class ParametersTest {
+
+    private static final Path CONFIG_PATH_DEFAULT = Paths.get("server/parameters/config.yml");
+
+    @Test
+    public void test_log_archive_age_limits() throws IOException, InterruptedException, TypeDBCheckedException {
```

Comment:
We just have one basic test for now, which checks that log archives are rolled over periodically and deleted after the configured age limit.